infra/main.bicep

```diff
@@ -200,3 +212,21 @@ output SERVICE_API_NAME string = api.outputs.SERVICE_API_NAME
 output SERVICE_WEB_NAME string = web.outputs.SERVICE_WEB_NAME
 output USE_APIM bool = useAPIM
 output SERVICE_API_ENDPOINTS array = useAPIM ? [ apimApi.outputs.SERVICE_API_URI, api.outputs.SERVICE_API_URI ]: []
+
+// API bindings common
+output AZURE_RESOURCE_GROUP string = rg.name
+
+// Source resource API
+// output SERVICE_API_NAME string = api.outputs.SERVICE_API_NAME
```

Comment:
Why not output it?

Reply:
ENV with same name exists at line 211, so it's duplicated

Reply:
I believe that is needed in that format for azd integration with aca, I'd keep all existing env vars, or do a full regression